translations/title_page.tex/de.po

```diff
@@ -18,13 +18,15 @@ msgstr ""
 
 #.  !TeX spellcheck = en_US
 #. type: tikzpicture
-#: sections/title_page.tex:13
+#: sections/title_page.tex:15
 #, no-wrap
 msgid ""
 "[remember picture, overlay, inner sep=10pt]\n"
-"  \\node(cover)[anchor=center] at (current page.center) {\n"
-"    \\includegraphics[height=\\paperheight, keepaspectratio]{\\layout/cover.jpg}\n"
-"  };\n"
+"  \\iftoggle{noartbackground}{}{\n"
```

Comment:
`msgstr`s need to be updated manually.

Reply:
How? This looks correct to me. What should I change here?

Reply:
Oh I see now

Reply:
Updated